corehq/apps/users/bulk_download.py

```diff
@@ -9,12 +9,12 @@
 from corehq import privileges
 from corehq.apps.accounting.utils import domain_has_privilege
 from corehq.apps.custom_data_fields.models import CustomDataFieldsDefinition
-from corehq.apps.es import UserES
 from corehq.apps.groups.models import Group
 from corehq.apps.locations.models import SQLLocation
 from corehq.apps.user_importer.importer import BulkCacheBase, GroupMemoizer
 from corehq.apps.users.dbaccessors.all_commcare_users import (
     get_commcare_users_by_filters,
+    get_mobile_usernames_by_filters,
```

Comment:
Nice, I did miss this on my first review.

Reply:
No worries, i just took the dedup as a followup so that it's clear.